molecule/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pip
+setuptools
+molecule[lint]
```

Comment:
Do you know if we can run these molecule tests in the github CI/travis?

Reply:
Currently working on that...

Reply:
Travis config merged